inc/agent.class.php

```diff
@@ -235,31 +235,68 @@ function showForm($id, $options = []) {
       if (!$this->isNewItem()) {
          echo "<tr class='tab_bg_1'>";
          echo "<td>"._n('Version', 'Versions', 1)."</td>";
-         echo "<td class='center'>";
+         echo "<td>";
          $versions = importArrayFromDB($this->fields["version"]);
          foreach ($versions as $module => $version) {
             echo "<strong>".$module. "</strong>: ".$version."<br/>";
          }
          echo "</td>";
          echo "<td>".__('Tag')."</td>";
-         echo "<td class='center'>";
+         echo "<td>";
          echo $this->fields["tag"];
          echo "</td>";
          echo "</tr>";
 
          echo "<tr class='tab_bg_1'>";
          echo "<td>".__('Useragent')."</td>";
-         echo "<td class='center'>";
+         echo "<td>";
          echo $this->fields["useragent"];
          echo "</td>";
          echo "<td>".__('Last contact')."</td>";
-         echo "<td align='center'>";
+         echo "<td>";
          echo Html::convDateTime($this->fields["last_contact"]);
          echo "</td>";
          echo "</tr>";
 
          echo "<tr class='tab_bg_1'>";
-         echo "<td colspan='2'>";
+         echo "<td>".__('Discovery threads')."</td>";
+         echo "<td>";
+         Dropdown::showNumber(
+            'threads_networkdiscovery', [
+               'value' => $this->fields['threads_networkdiscovery'],
+               'toadd' => [__('General setup')]
+            ]
+         );
+         echo "</td>";
+         echo "<td>".__('Discovery timeout')."</td>";
+         echo "<td>";
+         Dropdown::showNumber(
+            'timeout_networkdiscovery', [
+               'value' => $this->fields['timeout_networkdiscovery'],
+               'toadd' => [__('General setup')]
+            ]
+         );
+         echo "</td>";
+         echo "</tr>";
+
+         echo "<tr class='tab_bg_1'>";
+         echo "<td>".__('Inventory threads')."</td>";
```

Comment:
Maybe to just avoid any confusion, we should probably use "Network Inventory" or "NetInventory" in place of "Inventory" here. The same at l.292. The name should then be also changed for "Discovery" at l. 262 & l.271 to keep naming consistency.